### DISCUSSION TITLE
Welcome to Ollama-SwiftUI Discussions!

### DISCUSSION THREAD

**Discussion by kghandour** [kghandour/Ollama-SwiftUI#38] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by saamerm** [comment]
Great work Karim! Really appreciate you sharing this!
I also went ahead and shortened the code to a one page ContentView for easy use an integration without adding any packages:
https://gist.github.com/saamerm/bf20dff87f0173817f610290968638f3
